Update examples

LucieContamin opened the pull request:
- add metadata
- update target data format (aggregate into one parquet file format)
- update config files / article

elray1 reviewed (changes requested):
I added a couple of comments, and added a PR [here](https://github.com/Infectious-Disease-Modeling-Hubs/example-complex-scenario-hub/pull/14) with some minor changes to the text.

The last two plots didn't work for me -- example output below.  I used the `fix_21` branch of hubVis.  Any thoughts about what might be going on?

![image](https://github.com/Infectious-Disease-Modeling-Hubs/example-complex-scenario-hub/assets/5132208/7585e456-9ede-4a22-ad53-b3cb7edd25c0)

LucieContamin commented on hub-config/model-metadata-schema.json:
```diff
@@ -1,191 +1,144 @@
-  "properties": {
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Schema for Modeling Hub model metadata",
+    "description": "This is the schema for model metadata files, please refer to https://github.com/covid19-forecast-hub-europe/covid19-forecast-hub-europe/wiki/Metadata for more information.",
```
Thank you for noticing! I missed it. I will update it to the README in the example repo.

LucieContamin commented on model-metadata/hubcomp_example.yaml:
```diff
@@ -0,0 +1,21 @@
+team_name: "Hub Complex Example"
+team_abbr: "hubcomp"
+model_name: "Example Model output"
+model_abbr: "examp"
```
Thanks for catching that! I fix the file name!

LucieContamin commented:
@elray1 . Thank you very much for your review.

I update the metadata schema `description` information to update the link and fix the `.yaml` file name. Thank you again!

For the plots, it was cause by the empty `age_group` columns. In a past version of the file, I was removing it. So, I updated the code here to remove the empty columns, but I also updated the hubvis `fix_21` branch (PR [here](https://github.com/Infectious-Disease-Modeling-Hubs/hubVis/pull/23)) to automatically remove empty columns and returns a warning, to avoid that problem in the future.

elray1 reviewed (approved):
Thanks, this all looks good to me!
